Tests/Functional/RunnerTest.php

```diff
@@ -1963,4 +2026,64 @@ public function testTokenObfuscate()
         $this->assertNotContains(STORAGE_API_TOKEN, $output);
         $this->assertContains('[hidden]', $output);
     }
+
+    public function testPermissionsFailedWithoutContainerRootUserFeature()
+    {
+        $runner = $this->getRunner(new NullHandler());
+
+        $componentData = [
+            'id' => 'docker-demo',
+            'type' => 'other',
+            'name' => 'Docker Runner Test',
+            'description' => 'Testing Docker',
+            'data' => [
+                'definition' => [
+                    'type' => 'builder',
+                    'uri' => 'keboola/docker-custom-php',
+                    'tag' => 'latest',
+                    'build_options' => [
+                        'parent_type' => 'quayio',
+                        'repository' => [
+                            'uri' => 'https://github.com/keboola/docker-demo-app.git',
+                            'type' => 'git'
+                        ],
+                        'commands' => [],
+                        'entry_point' => 'mkdir /data/out/tables/mytable.csv.gz && '
+                            . 'chmod 000 /data/out/tables/mytable.csv.gz && '
+                            . 'touch /data/out/tables/mytable.csv.gz/part1 && '
+                            . 'echo "value1" > /data/out/tables/mytable.csv.gz/part1'
```

Comment:
del

Reply:
myslíš to echo? kuafix, mi tam chybí chmod na part1 :-(